templates/master/00-master/openstack/files/ipv6-config.yaml

```diff
@@ -0,0 +1,11 @@
+{{- if eq .IPFamilies "DualStack"}}
```

Comment:
I'm not sure you need or want this logic. You're only configuring ipv6 connections anyway, and you want this for any ipv6 connection regardless of whether it's dual stack, dual stack ipv6 primary, or single stack ipv6 primary. All three of those are separate cases in the IPFamilies logic: https://github.com/openshift/machine-config-operator/blob/49f32d46538e4387966a22aba703f5e579a37356/pkg/apis/machineconfiguration.openshift.io/v1/types.go#L120